src/test/java/com/epam/test_generator/controllers/AdminControllerSecurityTest.java

```diff
@@ -0,0 +1,143 @@
+import org.codehaus.jackson.map.ObjectMapper;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+
+import javax.servlet.Filter;
+
+import static org.mockito.Matchers.anyLong;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+
+@ContextConfiguration(classes = {WebConfig.class})
+@WebAppConfiguration
+public class AdminControllerSecurityTest {
+
+    private final LoginUserDTO loginUserDTO = new LoginUserDTO();
+    private final ChangeUserRoleDTO changeUserRoleDTO = new ChangeUserRoleDTO();
+
+    @Autowired
+    UserDAO userDAO;
+    @Autowired
+    PasswordEncoder encoder;
+    @InjectMocks
+    @Autowired
+    JwtAuthenticationProvider jwtAuthenticationProvider;
+    @Mock
+    private User user;
+    @Mock
+    private User passiveUser;
+    @Mock
+    private UserService userService;
+    @InjectMocks
+    @Autowired
+    private TokenService tokenService;
+
+    @Autowired
+    private WebApplicationContext context;
+
+    @Autowired
+    private Filter springSecurityFilterChain;
+
+    private MockMvc mvc;
+
+
+    @Before
+    public void setup() {
+        MockitoAnnotations.initMocks(this);
+        mvc = MockMvcBuilders
+                .webAppContextSetup(context)
+                .addFilters(springSecurityFilterChain)
+                .build();
+
+        loginUserDTO.setEmail("test@email.com");
+        loginUserDTO.setPassword("test");
+
+        changeUserRoleDTO.setEmail("admin@email.com");
+        changeUserRoleDTO.setRole("TEST_LEAD");
+
+        when(passiveUser.getEmail()).thenReturn("admin@email.com");
+        when(passiveUser.getRole()).thenReturn(new Role("GUEST"));
+
+        when(user.getEmail()).thenReturn("test@email.com");
+        when(user.getPassword()).thenReturn("test");
+        when(user.getId()).thenReturn(new Long(1));
+
+    }
+
+    @Test
+    public void getUsers_code200_statusOk() throws Exception {
```

Comment:
Rename test methods according to TAI code style. https://github.com/TAI-EPAM/TAI-Code-Style